plugins/copy-to-clipboard/prism-copy-to-clipboard.js

```diff
@@ -10,9 +10,23 @@
 	}
 
 	var Clipboard = window.Clipboard || undefined;
+	var _nativeClipboard;
+
+	try {
+		// Try using something ClipboardJS-specific
+		new Clipboard('').destroy();
+	} catch(e) {
+		// Existing Clipboard variable is not ClipboardJS.
+		// Store it so we can restore it later.
+		_nativeClipboard = Clipboard;
+		Clipboard = undefined;
+	}
 
 	if (!Clipboard && typeof require === 'function') {
 		Clipboard = require('clipboard');
+		if (_nativeClipboard) {
```

Comment:
We shouldn't need to restore here because `require('clipboard')` shouldn't change the global instance.

Reply:
True that!